glueviz/meta.yaml

```diff
@@ -14,26 +14,58 @@ build:
     - glue-deps = glue._deps:main
   osx_is_app: True
 
+
+install_requires = ['numpy>=1.9',
+                    'pandas>=0.14',
+                    'astropy>=1.3',
+                    'matplotlib>=1.4',
+                    'qtpy>=1.1',
+                    'setuptools>=1.0',
+                    'ipython>=4.0',
+                    'ipykernel',
+                    'qtconsole',
+                    'dill>=0.2',
+                    'glue-vispy-viewers>=0.6',
+                    'xlrd>=1.0',
+                    'h5py>=2.4']
```

Comment:
Oops sorry, I had copied it in there temporarily to make sure everything matched :blush: